src/main/java/app/gpx_animator/Configuration.java

```diff
@@ -251,15 +273,27 @@ public File getLogo() {
         return logo;
     }
 
+    public String getLogoPosition() {
+        return logoPosition;
+    }
+
+    public String getAttributionPosition() {
+        return attributionPosition;
+    }
+
+    public String getInformationPosition() {
+        return informationPosition;
+    }
+
     public String getPhotoDirectory() {
         return photoDirectory;
     }
 
     public Long getPhotoTime() {
         return photoTime;
     }
-
-    public Long getPhotoAnimationDuration() {
+  
+   public Long getPhotoAnimationDuration() {
```

Comment:
Just a simple formatting issue: The modified whitespace in front of the line makes it look like you changed four additional lines.